### DISCUSSION TITLE
Hi, i find some error in japanese moon stations.

### DISCUSSION THREAD

**Discussion by oneforallallforone** [Stellarium/stellarium#3965] [Translations]
Hi, i find some error in japanese moon stations. When sky culture language is Simplified Chinese language, the right star between "危宿" and "壁宿" its name is "室宿". "房宿" is wrong.
![Snipaste_2024-11-03_18-59-25](https://github.com/user-attachments/assets/7408faf9-22bc-4380-afaa-fa35b564530e)

**Comment by gzotti** [comment]
You can fix this via Transifex
